pkg/crc/machine/client/client_windows.go

```diff
@@ -6,8 +6,6 @@ import (
 
 	"github.com/code-ready/crc/pkg/crc/errors"
 	"github.com/code-ready/machine/drivers/hyperv"
-	"github.com/code-ready/machine/drivers/virtualbox"
-	"github.com/code-ready/machine/libmachine/drivers/plugin"
```

Comment:
@gbraad this is also used for `hyperv` so this shouldn't be removed. Even CI is failing because of it.

Reply:
```
	case "hyperv":
			plugin.RegisterDriver(hyperv.NewDriver("", ""))
```

is still there